tests/data/test_dataloader.py

```diff
@@ -79,23 +68,34 @@ def build_mock_ft_streaming_dataset(
         'response': 'Hello world3'
     }]
 
+    columns, data_format = get_columns_and_format(dataset, pretokenize,
+                                                  lambda x: x)
+
     output_path = os.path.join(data_path, split)
 
     with MDSWriter(columns=columns, out=output_path,
                    compression=None) as output_writer:
         for sample in dataset:
+
             if pretokenize:
                 sample = tokenize_formatted_example(sample, tokenizer=tokenizer)
-                sample_to_write = {}
-                for key in columns.keys():
-                    if use_bytes:
-                        sample_to_write[key] = np.asarray(sample[key]).tobytes()
-                    else:
-                        sample_to_write[key] = np.asarray(sample[key],
-                                                          dtype=np.uint32)
+                sample_to_write = {'turns': []}
+                # convert to bytes
```

Comment:
there is no bytes here right? cause we're just doing json

Reply:
Correct. Oversight from me. I'll remove that comment.